pippo-session-parent/pippo-session/src/main/java/ro/pippo/session/ClassFilter.java

```diff
@@ -0,0 +1,24 @@
+package ro.pippo.session;
+
+import java.util.ArrayList;
+
+/**
+ * @author idealzh
+ */
+public class ClassFilter {

```

Comment:
We don't need this class. Put this logic directly in `FilteringObjectInputStream`.